include/dgl/array.h

```diff
@@ -121,6 +148,19 @@ inline bool IsValidIdArray(const dgl::runtime::NDArray& arr) {
   return arr->ndim == 1 && arr->dtype.code == kDLInt;
 }
 
+/*!
+ * \brief Pack a padded tensor to a triplet of concatenated tensor, their lengths,
+ * and offsets.
```

Comment:
This looks like a quite complicated API. Could you describe more about the arguments and returns? Give an example in the docstring will be even better.